credential/store/src/main/java/org/wildfly/security/credential/store/CredentialStore.java

```diff
@@ -354,6 +354,17 @@ public void remove(String credentialAlias, Class<? extends Credential> credentia
         return spi.getAliases();
     }
 
+    /**
+     * Returns {@code Set<String>} of types stored in this store with given alias.
+     *
+     * @param credentialAlias to find the credentials types in the store
+     * @return {@code Set<String>} of all credential types stored in this store with given alias
+     * @throws UnsupportedOperationException when this method is not supported by the underlying credential store
+     */
+    public Set<String> getStoredTypesForAlias(String credentialAlias) throws UnsupportedOperationException {
```

Comment:
Just a small comment, to be consistent with the parameter name used when removing a credential, perhaps we could rename this method to `getCredentialTypesForAlias`.